src/azure-cli/azure/cli/command_modules/util/_help.py

```diff
@@ -45,3 +45,13 @@
 type: command
 short-summary: Upgrade Azure CLI and extensions
 """
+
+helps['demo'] = """
```

Comment:
Just curious is this a convention to add a demo command here?  The command is convenient for developers but also visible to users right？

Reply:
> Just curious is this a convention to add a demo command here?

No. We don't have such convention. The only existing usage of this pattern is `az self-test`:

https://github.com/Azure/azure-cli/blob/3c54715e05ccc656aaaa31a001485a02fa4ec424/src/azure-cli/azure/cli/command_modules/profile/__init__.py#L32

> The command is convenient for developers but also visible to users right？

It is not visible to the user, as the parent command group has been marked as `hide=True`:

https://github.com/Azure/azure-cli/blob/bc2d0d434fdd410dd510327f60dbd457778d08dd/src/azure-cli/azure/cli/command_modules/util/commands.py#L18-L19